Common/Sources/Extensions/HyperwalletTransferMethodType.swift

```diff
@@ -48,8 +48,8 @@ public extension HyperwalletTransferMethodType {
                                          processingTimeLabel,
                                          "add_transfer_method_processing_time_label".localized())
 
-            attributedText.append(value: processingTimeLabel, font: Theme.Label.captionOne, color: color)
-            attributedText.append(value: processingTimeValue, font: Theme.Label.captionOneMedium, color: color)
+            attributedText.append(value: processingTimeLabel, font: UIFont.preferredFont(forTextStyle: .caption1), color: color)
```

Comment:
same as above. Any changes made should be made in Theme class

Reply:
Removed preferredFont